modules/router-certificates.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/certificate-reference.adoc
+
+[id="router-certificates_{context}"]
+= Router certificates
+
+[discrete]
+== Purpose
+
+The Ingress Operator uses certificates for:
+
+* Securing access to metrics for Prometheus.
+* Securing access to Routes.
+
+[discrete]
+== File path
```

Comment:
Would it make sense to have the "File path" sections be named something like "Location" instead, since it's not always a file on the file system?